src/types/dashboard.ts

```diff
@@ -0,0 +1,15 @@
+//Can be expanded with visualisation method if needed.
+//Hva vi må legge til. Text. Title. Visualisation method. Query. //Source.
```

Comment:
The norwegian comment can be removed
```suggestion
// Can be expanded with visualisation method if needed.
```